app/views/projects/show.html.erb

```diff
@@ -1,20 +1,17 @@
-<%-# TODO: tables should be variable width depending on how many years of data we have -%>
-
 <h1><%= @project.name %></h1>
 
-<table>
-  <tr>
-    <th>Year</th>
-    <% @project.years.each do |year| %>
-      <th><%= year.date %></th>
-    <% end %>
-  </tr>
-  <% @project.years.each do |year| %>
-    <tr>
-      <td>dummy_attr</td>
-      <% @project.years.each do |year| %>
-        <td>dummy_value</td>
-      <% end %>
-    </tr>
+<h2>Project Details</h2>
+
+<% for attribute in @project.attributes.keys %>
```

Comment:
nit: In Ruby we don't use `for` loops. Instead, we use `@project.attributes.keys.each do |attribute| .... end`. Keep in mind for next time!